monitor/services/src/main/java/org/datacleaner/monitor/server/filesystem/WindowsWaitForCompleteFileStrategy.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.monitor.server.filesystem;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+/**
+ * Windows implementation of {@link WaitForCompleteFileStrategy}.
+ */
+public class WindowsWaitForCompleteFileStrategy extends AbstractWaitForCompleteFileStrategy {
+    /**
+     * It relies on a locking mechanism when a file being copied is locked from other processes.
+     */
+    @Override
+    public boolean isReady(final File file) {
+        try (FileInputStream inStream = new FileInputStream(file)) {
+            inStream.read();
+
+            return true;
+        } catch (final IOException ioe) {
```

Comment:
Let's just use `e`, It's not like catch blocks are large enough that anyone would be in doubt about which exception it is :)